### PR TITLE
Feature/remove bundle type adapter

### PR DESCRIPTION
**Pull request by lauriskr**
Remove the official support for storing Bundles since Bundle serialization and deserialization does not work properly and we don't want to support the feature.

### CODE REVIEW

File: labencryptedstorage/src/main/java/mobi/lab/labencryptedstorage/impl/KeyValueStorageEncryptedSharedPreferences.kt [hunk 1]
```diff
@@ -124,7 +122,7 @@ public class KeyValueStorageEncryptedSharedPreferences constructor(
         return when (encryptionPreferredTypeInternal) {
             EncryptionPreferredType.StrongBox -> SelectedStoragePersistenceId.ENCRYPTED_STRONG_BOX_PREFERRED
             EncryptionPreferredType.Tee -> SelectedStoragePersistenceId.ENCRYPTED_TEE_PREFERRED
-        }.exhaustive
```
Comment (harri35) [inline labencryptedstorage/src/main/java/mobi/lab/labencryptedstorage/impl/KeyValueStorageEncryptedSharedPreferences.kt:127]: Why did you remove the exhaustive support? So it doesn't conflict with integrating code? We should rename the method then maybe?

File: build.gradle [hunk 2]
```diff
@@ -10,8 +10,8 @@ allprojects {
 ext {
     // Version name will be versionMajor.versionMinor.versionPatch
     versionMajor = 0
-    versionMinor = 9
-    versionPatch = 6
+    versionMinor = 10
```
Comment (harri35) [inline build.gradle:13]: Please update the changelog also and add a note about the breaking change.

**Comment by harri35** [comment]
Just a comment - you can create an issue also under issues - less detailing is needed under the PR. And better history.

**Review by harri35 — APPROVED** [review]
Looks fine to me. It can be merged when comments are addressed.
Nice work!

File: labencryptedstorage/src/main/java/mobi/lab/labencryptedstorage/impl/KeyValueStorageEncryptedSharedPreferences.kt (hunk 1 above)
Comment (lauriskr) [reply labencryptedstorage/src/main/java/mobi/lab/labencryptedstorage/impl/KeyValueStorageEncryptedSharedPreferences.kt:127]: This is the default behaviour since Kotlin 1.7 and the extension is not needed anymore.

**Comment by lauriskr** [comment]
Created issue: https://github.com/LabMobi/LabEncryptedStorage/issues/7

File: build.gradle (hunk 2 above)
Comment (lauriskr) [reply build.gradle:13]: Updated the changelog
